proxy/proxymanager.go

```diff
@@ -387,6 +389,27 @@ func (pm *ProxyManager) unloadAllModelsHandler(c *gin.Context) {
 	c.String(http.StatusOK, "OK")
 }
 
+func (pm *ProxyManager) running(context *gin.Context) {
+	context.Header("Content-Type", "application/json")
+	response := gin.H{} // Default to an empty JSON object
+
+	for _, process := range pm.currentProcesses {
+		if process.config.Unlisted {
+			break // Stop checking if an unlisted process is found
```

Comment:
This endpoint is to expose the internal state of what's running. Excluding unlisted may lead to confusion when resources like VRAM is used but llama-swap shows nothing running.